nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -115,9 +122,13 @@ struct nxagentSpecialKeystrokeMap default_map[] = {
   {KEYSTROKE_IGNORE, ControlMask, True, XK_BackSpace},
   {KEYSTROKE_IGNORE, 0, False, XK_Terminate_Server},
   {KEYSTROKE_FORCE_SYNCHRONIZATION, ControlMask, True, XK_j},
+#ifdef DUMP
   {KEYSTROKE_REGIONS_ON_SCREEN, ControlMask, True, XK_a},
+#endif
+#ifdef NX_DEBUG_INPUT
   {KEYSTROKE_TEST_INPUT, ControlMask, True, XK_x},
   {KEYSTROKE_DEACTIVATE_INPUT_DEVICES_GRAB, ControlMask, True, XK_y},
+#endif
```

Comment:
> Plus: would it make sense to drop those actions from the new keystrokes enumerator (in Keystroke.h)?

Being more precise: ifdef'ing out the KEYSTROKE_<debug> enum items in Keystroke.h...